### DISCUSSION TITLE
Loop Mode with Max Lens Mod 2.0

### DISCUSSION THREAD

**Discussion by MtnXfreeride** [gopro/labs#600] [Feature Requests]
Could GoPro labs enable loop mode on the hero12 with max lens mod 2.0 in use?  I am curious why it isn't an option on a stock GoPro if anyone has that info as well.   I don't think overheating is an issue at 4k 30fps. 

Loop mode is popular for fishing and the lens mod allows a good wide angle view from behind on a kayak or on a life jacket it gives a horizon leveled wide view as well.

**Comment by dnewman-gpsw** [comment]
As Labs can't add video modes, this is not potential Labs feature. Looping would need to be added to the main firmware.  A lot of fishing video is done using Hindsight, which is offered in Max Lens Mod modes.

**Reply by MtnXfreeride** [reply]
Hindsight is almost usable but current state it breaks timecode once you go past 30 seconds.   

Also, 30 seconds is not quite enough for fishing because it means you need to take your hand off the rod to push a button when you are at the highest risk of losing the fish.   1 minute would be perfect to catch the cast, the hook, and highly likely to have it in a net or on boat within a minute.  

Is extending hindsight possible with labs, that could be perfection to have 1 to 2 mins instead of 30 seconds max.  

Hindsight has a higher overheating risk than loop mode in my testing but I'll give it a try once the GoPro labs fix is in for the timeout.   If I set all 3 gopros on hindsight the timecode issue might not matter.
